Resources/public/js/models/ez-locationmodel.js

```diff
@@ -180,6 +180,21 @@ YUI.add('ez-locationmodel', function (Y) {
         },
 
         /**
+         * Swap the location with other location
+         * 
+         * @method swap
+         * @param {Object} options the required for the swap         *
+         * @param {Object} options.api (required) the JS REST client instance
+         * @param {eZ.Location} destination Location where the given one will be relocated
```

Comment:
`@param {eZ.Location} destinationLocation Location where the given one will be relocated`